Makefile

```diff
@@ -51,54 +52,63 @@ _ := $(foreach exec,$(REQ_BINS), \
        $(if $(shell which $(exec)),some string,$(error "No $(exec) binary in $$PATH")))
 
 
-all:        build pack release-artifacts                            ## Clean, build and pack
+## Clean, build and pack
+all: build pack release-artifacts
 .PHONY: all
 
-help:                                                               ## Prints list of tasks
-	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z0-9_-]+:.*?## / {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}' Makefile
+## Prints list of tasks
+help:
```

Comment:
make target commans have to start with tabs, otherwise:
```
$ gmake help                                                                                                                                                                   0
gmake: Nothing to be done for 'help'.
```

Reply:
Let me know if it works now ;)

Reply:
Nope :( there still seems to be a space (instead of tab) in front of the `  @awk ...` line

Reply:
honestly this feels alittle insane, makefile feels super sensitive.

Reply:
I think it's just Make syntax (always been :)), seems like it's configurable in GNU make with `.RECIPEPREFIX` [1], still will take only 1 char 😄 . I agree it's a bit odd, but this would bring to a long discussion "Yeah Make is not that great"... but what is better and present everywhere by default? :)

[1]: https://www.gnu.org/software/make/manual/html_node/Rule-Syntax.html

Reply:
I think its best if we just print out the help details, rather than parsing our own makefile.